### DISCUSSION TITLE
Bug report: Scrolling past image prematurely unloads username

### DISCUSSION THREAD

**Discussion by Moosems** [community/community#67527] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

If I scroll past a certain point the username of people disappears even though I did not scroll past the viewable area it was in.

Video:

https://github.com/community/community/assets/95927277/09817571-8d8d-4213-a538-d4361a75a72e

**Comment by hafiz-muhammad** [comment]
That is by design, it isn't a bug.

**Reply by Moosems** [reply]
Why would they do that? Kinda unnecessary imo.
